src/ResourceManager/Cdn/Commands.Cdn/Models/Profile/PSProfileResourceState.cs

```diff
@@ -0,0 +1,26 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.Cdn.Models.Profile
+{
+    public enum PSProfileResourceState
```

Comment:
I think enum is more accurate.
One version of powershell is corresponding to one version of sdk, one version of SDK is corresponding to one version of API. Since SDK have this value as enum, powershell should be consistent.